operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -77,6 +77,8 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.
```

Comment:
Propose we include an example. It is not clear to a new reader why using second person doesn't keep the audience focused the same way as using first person.

Reply:
Here's an example from the *Mobile App Deployment Guide*

>A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
>
>We’re here to support you.

With the current doc guidelines this might sound something like this

> Deploying the Mattermost Mobile App will take time and iteration.

The same content is delivered but the first leaves the reader with the comfort of companionship where the second might stop the reader from trying it at all.

Would this be a suitable example?

Reply:
@jasonblais from our chat yesterday I refined the example:

> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.
>
>We are here to [support](link-to-support-section) you.

As opposed to

> Deploying the Mattermost Mobile App will take time and iteration.

Reply:
@faase The refined example makes sense to me as an outside reader.  Recommend incorporating it into the piece.